api.js

```diff
@@ -489,8 +489,8 @@ async function getState() {
 async function getDelayedState() {
   let blockNumber = await web3.eth.getBlockNumber();
   // its too late to make the tx
-  if (blockNumber % 10 > 4 || blockNumber % 10 < 1) return -1;
-  let state = Math.floor(blockNumber / 10);
+  if (blockNumber % 75 > 4 || blockNumber % 75 < 1) return -1;
```

Comment:
Shouldn't the buffer period for every state also increase?
It was `(blockNumber % 10 > 4 || blockNumber % 10 < 1)` when we were calculating mod 10
But since we are calculating mod 75 now, >4 and <1 would be a very small buffer period.

Reply:
@SkandaBhat I would like to know your thoughts too.